docs/site/package.json

```diff
@@ -27,6 +27,8 @@
     "object-assign": "^4.1.0",
     "raw-loader": "^0.5.1",
     "react-hot-loader": "^3.0.0-beta.2",
+    "react-motion": "^0.4.5",
```

Comment:
Can't we use the `material-ui` slide transition? I'm more than happy to make any required changes to support the use case.